input/fsh/profiles-resources/CHCoreImmunization.fsh

```diff
@@ -0,0 +1,50 @@
+Profile: CHCoreImmunization
+Parent: Immunization
+Id: ch-core-immunization
+Title: "CH Core Immunization"
+Description: "CH Core definition of the immunization."
```

Comment:
Vorschlag analog Medication:
Base definition of the Immunization resource for use in Swiss specific use cases.